src/components/scenes/CryptoExchangeQuoteProcessingScene.tsx

```diff
@@ -61,27 +64,39 @@ export function CryptoExchangeQuoteProcessingScene(props: Props) {
 
   return (
     <SceneWrapper>
-      <View style={styles.container}>
-        <EdgeText style={styles.title}>{lstrings.hang_tight}</EdgeText>
-        <EdgeText style={styles.findingText} numberOfLines={3}>
-          {isLongWait ? lstrings.exchange_slow : lstrings.trying_to_find}
-        </EdgeText>
-        <ActivityIndicator style={styles.spinner} color={theme.iconTappable} />
-      </View>
-      {!isLongWait ? null : (
-        <ButtonsViewUi4
-          absolute
-          fade
-          primary={{
-            label: lstrings.string_cancel_cap,
-            onPress: () => {
-              mounted.current = false
-              onCancel()
-            }
-          }}
-          layout="column"
-          sceneMargin
-        />
+      {({ insetStyle }) => (
+        <View style={{ paddingBottom: insetStyle.paddingBottom, width: '100%', height: '100%' }}>
```

Comment:
Why the extra view instead of using the one with `styles.container`?

Also, why only use `paddingBottom` from insetStyle? Using all of the rules usually means we're avoiding insets on left/right and even top sides of the device (e.g. for rotated iPad devices)

Reply:
The extra View covers all the content including the button at the bottom while the inner View is just the text and spinner.

I didn't actually need the insets at all since all the content is centered. Removed.